neuralg/ops/utils/predict.py

```diff
@@ -1,11 +1,10 @@
-import neuralg
+from ... import neuralg_SAFEMODE
 import torch
 from loguru import logger
-from ...utils.set_log_level import set_log_level
 
 
 def predict(model, input):
-    """ Predicts output with model on passed input with potential safety checks on outputs
+    """Predicts output with model on passed input with potential safety checks on outputs
```

Comment:
are you using black yet? :P if not it may be nice to have git-hook to autoformat https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks code with https://github.com/psf/black

Reply:
Yeees I should be using black not sure how this sneaked its way in, but good idea to automate it. Will look into!